pkg/cvo/cvo_scenarios_test.go

```diff
@@ -94,6 +94,7 @@ func setupCVOTest(payloadDir string) (*Operator, map[string]runtime.Object, *fak
 		},
 		"exclude-test",
 		record.NewFakeRecorder(100),
+		"default",
```

Comment:
This should get set in `&Operator` above and then passed through here.

Reply:
done :+1: